docs/expressions/arithmetic.md

```diff
@@ -184,21 +176,21 @@ This allows more aggressive optimizations and for faster execution, but only yie
 
 ---
 
-Operator | Method
----------|---------------
-`+~`     | add_unsafe()
-`-~`     | sub_unsafe()
-`*~`     | mul_unsafe()
-`/~`     | div_unsafe()
-`%~`     | rem_unsafe()
-`%%~`    | mod_unsafe()
-`-~`     | neg_unsafe()
-`<~`     | lt_unsafe()
-`>~`     | gt_unsafe()
-`<=~`    | le_unsafe()
-`>=~`    | ge_unsafe()
-`=~`     | eq_unsafe()
-`!=~`    | ne_unsafe()
+| Operator | Method       |
```

Comment:
this looks a bit more irritating to maintain.

Reply:
Are you talking about the additional `|` before and after all tables? I did not realize that was happening as it is part of my VS Code setup to run GFM table formatting on save.

No real opinion on reversing this as it was not intentionally done in the first place,

Reply:
Ah, I assumed it was required. Yeah, not having to have is nice.